src/apiCalls.js

```diff
@@ -0,0 +1,19 @@
+export const getUser = async (email, password) => {
+  const url = 'https://rancid-tomatillos.herokuapp.com/api/v1/login';
+  const options = {
+    method: 'POST',
+    body: JSON.stringify({
+      email,
+      password
+    }),
+    headers: {
+      'Content-Type': 'application/json'
+    }
+  }
+  const res = await fetch(url, options)
+  if (!res.ok) {
+    throw new Error("Error! No 200 Status Code Found.")
```

Comment:
We might want to make this error something that displays to the user later and word it so that they have a better idea of what went wrong

Reply:
Error is temporary boilerplate. We will refactor. 👍